packaging/linux/postinstall

```diff
@@ -1,15 +1,21 @@
 #!/bin/bash
 
+set -o errexit
+
 user=$(loginctl list-users --output json | jq -r ".[0].user")
 home=$(getent passwd "${user}" | cut -d: -f 6)
 
 config_dir="${home}/.config/naisdevice/"
-unit_file=/lib/systemd/system/device-agent-helper.service
+unit_file=/usr/lib/systemd/system/device-agent-helper.service
 
 mkdir -p "${config_dir}"
 
 sed -i "s%@@CONFIG_DIR@@%${config_dir}%" "${unit_file}"
+systemctl daemon-reload
+systemctl restart device-agent-helper
 
 cp /sys/devices/virtual/dmi/id/product_serial "${config_dir}"
 
 chown -R "${user}:" "${config_dir}"
+
+update-desktop-database
```

Comment:
I think dpkg already does this, via triggers:

```
Processing triggers for desktop-file-utils (0.24-1ubuntu4) ...
```

Reply:
I didn't need to do `update-desktop-database` for naisdevice to show up atleast.

Reply:
Alright, good. I just confused this issue with the non executeable binaries